build/Dockerfile

```diff
@@ -23,14 +23,15 @@ ENV LC_CTYPE=ru_RU.UTF-8
 RUN pip3 install -U pip && \ 
     hash -r pip3 && \
     pip3 install -U setuptools && \
+    apt-get install -y python3-venv && \
     pip3 install pipenv && \
+    pip3 install virtualenv && \
```

Comment:
`pipenv` requires `virtualenv`, so there is no point to have virtualenv after pipenv